go/client/util.go

```diff
@@ -0,0 +1,26 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/keybase/client/go/libkb"
+)
+
+func ensureSetPassphraseFromRemote(mctx libkb.MetaContext) error {
+	if !mctx.G().ActiveDevice.Valid() {
+		return nil
+	}
+	cli, err := GetUserClient(mctx.G())
+	if err != nil {
+		return err
+	}
+	ret, err := cli.CanLogout(mctx.Ctx(), 0)
```

Comment:
btw this already does the `!h.G().ActiveDevice.Valid()` call

Reply:
good catch, i'll remove it here then